controller/setting_controller.go

```diff
@@ -254,6 +265,29 @@ func (sc *SettingController) syncBackupTarget() (err error) {
 	return nil
 }
 
+func (sc *SettingController) updateBackupstorePollInterval() (err error) {
+	if sc.bsMonitor == nil {
+		return nil
+	}
+
+	defer func() {
+		err = errors.Wrapf(err, "failed to sync backup target")
+	}()
+
+	interval, err := sc.ds.GetSettingAsInt(types.SettingNameBackupstorePollInterval)
+	if err != nil {
+		return err
+	}
+
+	sc.bsMonitor.Stop()
```

Comment:
Check if the interval was the same as before. Even without modification, the trigger will be called once in a while.